config/templates/postgresql-db.yaml

```
@@ -235,13 +351,13 @@ spec:
               name: postgres-operator-cluster-environment
         s3Bucket:
           - description: '{{.metadata.name}} (backup)'
-            bucket: '{{ kget (print "secret/postgres-operator/backup-postgres-" .metadata.name "-config") "LOGICAL_BACKUP_S3_BUCKET" }}'
+            bucket: '{{.spec.backup.bucket}}'
```

```suggestion
            bucket: '{{ .spec.backup.bucket | default (kget "secret/postgres-operator/postgres-operator-cluster-environment" "BACKUP_S3_BUCKET") }}'
```
to match the restic definition and allow safe handling of the case where backup.bucket is not specified

---

This was the original setup prior to my restic implementation so I just revert it to the original one. But yeah I think adding it back should not hurt. Let me add this back.

---

Ah, sorry I read too fast. You were suggesting add an improvement, not reverting to the old code. Thanks for the improvement. I added your suggestion!